examples/sampling-integrated-gaussian-log-likelihood.ipynb

```diff
@@ -0,0 +1,542 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Integrating out the noise in Gaussian likelihood problems\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "A property of Gaussian likelihoods and certain is that the joint posterior distribution $p(\\theta,\\sigma)$ can be analytically integrated with respect to $\\sigma$ to yield the marginal distribution $p(\\theta)$. For example, for Gaussian log-likelihood models where we assume a prior, $\\sigma\\sim U(a, b),$ then the marginal posterior is given by,\n",
+    "\n",
```

Comment:
This seems incredibly niche. When did we get interested in sampling from gaussian likelihoods that we can analytically analyse? Sorry, really not getting an idea of the application from this description. For which kind of problems can we expect a suitable loglikelihood to arise, and what advantages does this give us?

Reply:
See the docstrings. This is exactly the same posterior distribution as we have in all the current example model notebooks except we have done some analytical integration so that we don't have to infer sigma (when sigma is set a uniform prior). Having one fewer parameters may speed up sampling.